libs/platform/offline/src/services/network-state-default.service.ts

```diff
@@ -0,0 +1,22 @@
+import {
+  Observable,
+  distinctUntilChanged,
+  fromEvent,
+  map,
+  merge,
+  of,
+} from 'rxjs';
+import { NetworkStateService } from './network-state.service';
+
+export class NetworkStateDefaultService implements NetworkStateService {
+  online(): Observable<boolean> {
+    return merge(
+      of(null),
```

Comment:
why do we need it?

Reply:
we need initial value for the first emit, due to window is emitting 'offline' | 'online' events only when state really changes

Reply:
use startWith instead then